src/components/header/header.js

```diff
@@ -27,7 +31,10 @@ const Header = () => {
                         </HeaderLink>
                     )
                 })}
-                <HeaderWork>Work with us</HeaderWork>
+                <HeaderWork
+                    onClick={openPopup}>
```

Comment:
if you put onClick={props.openPopup} you don't need the duplicate function at the top of this file